api/model.go

```diff
@@ -1272,6 +1290,7 @@ func toVolumeResource(v *longhorn.Volume, ves []*longhorn.Engine, vrs []*longhor
 			actions["updateReplicaAutoBalance"] = struct{}{}
 			actions["updateUnmapMarkSnapChainRemoved"] = struct{}{}
 			actions["updateSnapshotDataIntegrity"] = struct{}{}
+			actions["updateBackupCompressionMethod"] = struct{}{}
```

Comment:
Just double confirm: Because the compression method will be saved for each backup, backups with different compression method before and after, those backups should be able to restore correctly, right?

Reply:
For a volume, the backups use the same compression method in the current implementation. Changing of the volume's compression method is not supported and rejected by the validator.

https://github.com/longhorn/longhorn-manager/blob/5ab00441dd915c94f0ce10ca1a7c74212127679c/webhook/resources/volume/validator.go#L149

We can improve it if users have the request.